FrejaEidClient/src/main/java/com/verisec/frejaeid/client/enums/AttributeToReturn.java

```diff
@@ -71,7 +72,12 @@
      * requested only for {@linkplain MinRegistrationLevel#EXTENDED} and
      * {@linkplain MinRegistrationLevel#PLUS}.
      */
-    ADDRESSES("ADDRESSES");
+    ADDRESSES("ADDRESSES"),
+    /**
+     * If ALL_EMAIL_ADDRESSES is requested, list of {@linkplain Email} which contains all
```

Comment:
This sentence is a little bit weird, maybe to finish it with "will be returned".  Also update for email address that it is primary email.

Reply:
Done.